src/stories/availability-label/AvailabilityLabel.tsx

```diff
@@ -0,0 +1,50 @@
+import { Pagefold, PagefoldProps } from "../pagefold/Pagefold";
+
+export type AvailabilityLabelPropsType = {
+  manifestation: "bog" | "ebog" | "lydbog (net)" | "lydbog (cd-mp3)";
+  availability: "Hjemme" | "Online" | "Udlånt";
```

Comment:
Is there a reason behind the capitalization here? I mean "bog" is all lower case while "Hjemme" has initial upper case.

In general I would expect text to have a natural casing (here initial upper case) and if special casing was required by the design then we would modify it using CSS.

Reply:
In regards to the capitalization - I wasn't sure what the data that would look like when we get if from the API, so I just made it random and not the same. 😅 I'll change it to be consistent 🤝